brewtils/models.py

```diff
@@ -530,6 +530,7 @@ class RequestTemplate(BaseModel):
         "command_type",
         "parameters",
         "comment",
```

Comment:
Hidden should only be on the Request object since it is added when we evaluate the Request for processing